Bynder/Api/Impl/Oauth/OauthRequestSender.cs

```diff
@@ -51,7 +52,9 @@ public OauthRequestSender(Credentials credentials, string baseUrl)
             _baseUrl = baseUrl;
             _httpClient = new HttpClient(new OAuthMessageHandler(credentials, new HttpClientHandler()));
             _httpClient.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
-        }
+			// Set the security protocol used by the ServicePoint
+			ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12 | SecurityProtocolType.Tls11;
```

Comment:
I don't think we need this as we are targetting .net46 and that is the default. Also, identation is not right.

Reply:
Starting with the .NET Framework 4.7, the default value of this property is SecurityProtocolType.SystemDefault and for versions of the .NET Framework through the .NET Framework 4.6 no default value is listed for this property.

https://docs.microsoft.com/en-us/dotnet/api/system.net.servicepointmanager.securityprotocol?view=netframework-4.7#System_Net_ServicePointManager_SecurityProtocol